src/Build/BackEnd/Components/Communications/NodeProviderOutOfProcTaskHost.cs

```diff
@@ -698,6 +696,18 @@ internal bool CreateNode(HandshakeOptions hostContext, INodePacketFactory factor
                 1);
 
             return nodeContexts.Count == 1;
+
+            // Determines whether node reuse should be enabled for the given host context.
+            // Node reuse allows MSBuild to reuse existing task host processes for better performance,
+            // but is disabled for CLR2.
+            bool EnableNodeReuse(HandshakeOptions hostContext)
```

Comment:
nit: I'd prefer a name like `NodeReuseIsEnabled` or `EnablesNodeReuse`--this sounds like an action, rather than a question.